app/views/publications/show.html.erb

```diff
@@ -1,51 +0,0 @@
-<% page_title edition_page_title(@document), t('publications.heading') %>
-
-<%= content_tag_for :article, @document, nil, class: "document-page #{@document.type.downcase}" do %>
-  <header class="block headings-block">
-    <div class="inner-block floated-children">
-      <%= render partial: 'header',
-                 locals: {
-                   header_title: @document.display_type,
-                   document: @document,
-                   footer_meta: true,
-                   policies: @related_policies
-                 } %>
-    </div>
-  </header>
-
-  <div>
-    <div class="inner-block">
-      <%= render partial: "document_summary", locals: { document: @document } %>
-    </div>
-  </div>
-
-  <div class="block-2 heading-block">
-    <div class="inner-block">
-      <%= render partial: "documents/attachment_full_width",
```

Comment:
Can this partial also be removed? Looks like it was only referenced on publications and statistics.